pysatMadrigal/tests/test_methods_general.py

```diff
@@ -502,8 +502,20 @@ def teardown_method(self):
         # Remove the temporary directory and file
         for tfile in self.temp_files:
             if os.path.isfile(tfile):
-                os.remove(tfile)
-        self.data_path.cleanup()
+                if hasattr(tfile, 'close'):
+                    tfile.close()
+
```

Comment:
```suggestion
```

This line is never reached.  https://coveralls.io/builds/61025359/source?filename=pysatMadrigal%2Ftests%2Ftest_methods_general.pyhttps://coveralls.io/builds/61025359/source?filename=pysatMadrigal%2Ftests%2Ftest_methods_general.py